new-pipeline/src/dashboards.js

```diff
@@ -1081,6 +1081,24 @@ function getDescriptionLink(metric, channel, description) {
       class: "btn btn-outline-primary fa fa-info-circle",
     }).text(" More details"));
   }
-
+  
   return link;
 }
+
+function getUseCounterLink(metric, channel, description) {
+  var metricUrl = buildDictionaryURL(metric, channel, description);
+  if (!metric.startsWith("USE_COUNTER2")) {
+    return null; // Clear use counter link;
+  }
+  //Show correct use counter link based on group selection.
+  var metricSplit = metric.split("_");
+  var group = metricSplit[2];
```

Comment:
`metricSplit` may have fewer than 3 entries. Let's add a check and early return?